github-dark.css

```diff
@@ -785,7 +785,7 @@
   .drag-and-drop, .octofication .broadcast-icon-mask, .file .image,
   .merge-pr-more-commits, .release-timeline-tags .expander-dots,
   .coupons .coupon-form-body, .manage-repo-access-group, .feature-callout,
-  .saved-reply-form, .timeline-progressive-disclosure-button {
+  .saved-reply-form, .timeline-progressive-disclosure-button, .next-charge-box {
```

Comment:
Should probably be `#1d1d1d`, given that it's a shade lighter than white originally. I'd make the border `#343434`.